packages/office-ui-fabric-react/src/components/FloatingPicker/BaseFloatingPicker.tsx

```diff
@@ -221,7 +216,9 @@ export class BaseFloatingPicker<T, P extends IBaseFloatingPickerProps<T>>
   }
 
   protected onChange(item: T): void {
-    if (this.props.onChange) {
+    const currentRenderedQueryString = this.props.currentRenderedQueryString;
+    const queryString = this.state.queryString;
+    if (this.props.onChange && currentRenderedQueryString !== undefined && currentRenderedQueryString === queryString) {
```

Comment:
Since `currentRenderedQueryString?: string;` is optional, won't this new prop break Users of the component that don't supply it? It will be `undefined` and therefore `this.props.onChange` will never be called.